packages/cursorless-engine/src/actions/Clear.ts

```diff
@@ -1,4 +1,4 @@
-import { PlainTarget } from "../processTargets/targets";
+import { PlainTarget } from "../processTargets/targets/PlainTarget";
```

Comment:
As discussed, you should only switch from index file to direct import when both files are descendants of the folder containing the index file. This import is likely what is breaking the build, because it subverts the barrel hack that we have to serialize the imports in target dir